README.md

```diff
@@ -23,10 +23,9 @@ Usage
 ----------
 
 ```
-from tinysegmenter import TinySegmenter
-segmenter = TinySegmenter()
+from tinysegmenter import tokenize
```

Comment:
In that case, you don't use class, using namespace would be better for avoiding name collision

```py
import tinysegmenter
```

and

```py
tokenized_statement = tinysegmenter.tokenize(satement)
```